src/main/java/core/garage/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.garage;
+
+import java.sql.SQLOutput;
+
+public abstract class Machine {
+    public abstract void doWork();
+    public abstract void stopWork();

```

Comment:
```suggestion
    public abstract void doWork();
    
    public abstract void stopWork();

```
при оголошенні між методами повинна бути пуста строка
спробуй запустити локально mvn clean package, щоб отримати ріпорт по код стайлу